docs/buildah-mount.md

```diff
@@ -4,9 +4,7 @@
 buildah\-mount - Mount a working container's root filesystem.
 
 ## SYNOPSIS
-**buildah** **mount**
-
-**buildah** **mount** [**containerID** [...]]
+**buildah** **mount** [**container** ...]
```

Comment:
Remove middle ** **

Reply:
> Remove middle ** **

Done, here and in `buildah-from.md`, with 1a6e426 -> 4a2b66a.